dist/images/Dockerfile.base

```diff
@@ -25,6 +25,7 @@ RUN cd /usr/src/ && git clone -b branch-21.06 --depth=1 https://github.com/ovn-o
     cd ovn && \
     curl -s https://github.com/kubeovn/ovn/commit/e24734913d25c0bffdf1cfd79e14ef43d01e1019.patch | git apply && \
     curl -s https://github.com/kubeovn/ovn/commit/8f4e4868377afb5e980856755b9f6394f8b649e2.patch | git apply && \
+    curl -s https://github.com/kubeovn/ovn/commit/8a557ebc313593e4392c246b512cc1c227b9fba9.patch | git apply && \
```

Comment:
`char *match = xasprintf("ip && ip4.dst == %s", svc_ipv4_cidr);`

`match` should be freed.